xlml/utils/xpk.py

```diff
@@ -37,98 +40,122 @@ def generate_workload_id(benchmark_id: str) -> str:
   return f"{short_benchmark}{short_id}"
 
 
-def _run_workload(
+@task
+def run_workload(
```

Comment:
Thanks for this change! You tried it on both local and composer env right? I previously tried to remove `kubernetes` dependency, but met one issue (sorry I did not recall what it was). So would like to double check. 

Could you check if any dependency from https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/master/.github/requirements.txt could be removed? If so, could you also help remove that from terraform template (if needed)? 

i.e.
```
apache-airflow-providers-docker
...
```

Reply:
Sure! I can remove that one, do you know if `kubernetes` is still required?

Reply:
Good question! Could you double check with the IDE and Pylint logs (if you run code-style.sh). If no import is in grey, I think we should be fine.

Reply:
I see, that makes sense. I don't see any usage of KubernetesPodOperator or task.kubernetes, so I'll remove it and see if the DagCheck passes. At any rate, there's no harm in keeping it since it's part of the Composer env.

Reply:
Looks like DAG check is green!

Reply:
We still need the `kubernetes` package for the GKE/GPU tests

Reply:
That's just `kubernetes` and not the Airflow package though, right? This is the one to be removed: `apache-airflow-providers-cncf-kubernetes`

Reply:
Yeah, removing the kubernetes airflow plugin is fine.